src/app/views/workflow-preview/WorkflowPreview.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import { push } from "react-router-redux";
+import { Link } from "react-router";
+import PropTypes from "prop-types";
+
+import url from "../../utilities/url";
+import notifications from "../../utilities/notifications";
+
+import Iframe from "../../components/iframe/Iframe";
+
+const propTypes = {
+    location: PropTypes.PropTypes.shape({
+        pathname: PropTypes.string
+    }),
+    params: PropTypes.shape({
+        collectionID: PropTypes.string.isRequired
+    }),
+    dispatch: PropTypes.func.isRequired
+};
+
+export class WorkflowPreview extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    handleBackButton = () => {
+        const previousUrl = url.resolve("../");
+        this.props.dispatch(push(previousUrl));
+    };
+
+    // getPreviewIframeURL returns the url for the content to be reviewed when the url is
+    // in the format: /florence/collections/collection-id/{content/to/preview/url}/preview
+    // returns '/' incase the url is '/homepage' or errors
+    getPreviewIframeURL = path => {
+        try {
+            const regex = /(?:\/[^/]+){3}(?<contentURL>.+)\/preview/;
```

Comment:
Wonder if we could give this a more meaningful name? Regex has never been a strong point of mine, so having a name that conveys what this pattern actually corresponds to could be helpful - possibly in particular when glancing through the code in the future